packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -100,15 +109,30 @@ export const iconMetadata: {
     IconBookMagnifyingGlass: {
         tags: ['analyze']
     },
-    IconCalendar: {
-        tags: ['schedule']
-    },
     IconCalendarCheckLines: {
         tags: ['test', 'plan', 'schedule']
     },
+    IconCalendarClock: {
+        tags: ['time', 'schedule']
+    },
     IconCalendarDay: {
         tags: ['date', 'time', 'schedule']
     },
+    IconCalendarDays: {
+        tags: ['schedule']
+    },
+    IconCalendarLines: {
+        tags: ['agenda']
+    },
+    IconCalendarRectangle: {
+        tags: []
+    },
+    IconCalendarToday: {
```

Comment:
I may have missed it but I don't see [calendar today](https://fontawesome.com/search?q=calendar%20today&o=r) in font awesome so this seems like a different `calendar-day` style.

What about `calendar-day-outline` with a tag of `today` as the metaphor? @fredvisser

Reply:
Yeah, I waffled between those 2 naming schemes - I'm good with `calendar-day-outline` and tag of `today` as the metaphor